policyengine_us/parameters/gov/states/az/hhs/tanf/eligibility/payment_standard/max_household_size.yaml

```diff
@@ -0,0 +1,11 @@
+description: Arizona caps the number of household members for which additional cash assistance amount can be received at the following.
```

Comment:
the program document doesn't say this. it might just not show those above 12. remove unless the cap is documented